cog/core/sql_abstract.py

```
@@ -0,0 +1,170 @@
+        uid: int,
+    ):
+
+        data = fetchone_by_primary_key("user", "uid", uid)
+        if data is None:
+            return None
+
+        record = UserRecord(
+            _protected=True, **data  # pyright: ignore[reportArgumentType]
+        )
+
+        assert any(
```

Ruff (0.8.2)

Use of `assert` detected

(S101)